packages/private-build-infra/src/create-version-module.js

```diff
@@ -28,6 +28,12 @@ function calculateVersion() {
   return packageVersion + suffix;
 }
 
-module.exports = function() {
-  return createFile('version.js', 'export default "' + calculateVersion() + '";');
+module.exports = function(compatVersion) {
+  return createFile(
+    'version.js',
+    'export default "' +
+      calculateVersion() +
+      '";\n' +
+      (compatVersion && compatVersion !== 'null' ? `export const COMPAT_VERSION = "${compatVersion}";\n` : '')
```

Comment:
This is intentionally checking for the string "null"? Seems slightly odd, why is that?

Reply:
left over from my initial implementation of 

```
  const compatWith = process.env.EMBER_DATA_FULL_COMPAT ? '99.0' : null;
```